docs/sphinx/conf.py

```diff
@@ -100,17 +100,19 @@
 if "SOURCE_BRANCH" in os.environ and len(os.environ.get('SOURCE_BRANCH')) > 0:
     branch = os.environ.get('SOURCE_BRANCH')
 else:
-    branch = 'develop'
+    branch = 'master'
 
 if "SOURCE_USER" in os.environ and len(os.environ.get('SOURCE_USER')) > 0:
-    user = os.environ.get('SOURCE_USER')
+    ome_user = os.environ.get('SOURCE_USER')
+    openmicroscopy_user = os.environ.get('SOURCE_USER')
 else:
-    user = 'openmicroscopy'
+    ome_user = 'ome'
+    openmicroscopy_user = 'openmicroscopy'
 
 github_root = 'https://github.com/'
-omero_github_root = github_root + user + '/openmicroscopy/'
-bf_github_root = github_root + user + '/bioformats/'
-doc_github_root = github_root + user + '/ome-documentation/'
+bf_github_root = github_root + openmicroscopy_user + '/bioformats/'
+model_github_root = github_root + ome_user + '/bioformats/'
+files_github_root = github_root + ome_user + '/ome-files/'
 
```

Comment:
I've removed ome-files and fixed ome-model.  The model root is here for future documentation source references.